presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/ReadPreferenceType.java

```diff
@@ -25,6 +25,7 @@
     SECONDARY_PREFERRED(ReadPreference.secondaryPreferred()),
     NEAREST(ReadPreference.nearest());
 
+    @SuppressWarnings("ImmutableEnumChecker")
     private final ReadPreference readPreference;
```

Comment:
We should mark `ReadPreference` as immutable

Reply:
The `Immutable` can only be annotated in class level.